README.md

```diff
@@ -67,15 +67,17 @@ Make sure you update the section in `project.clj` that looks like this to contai
 :dev [:qa {
   :env ^:replace {
     :open-company-auth-passphrase "this_is_a_dev_secret" ; JWT secret
-    :hot-reload "true" ; reload code when changed on the file system
     :open-company-slack-client-id "FIXME"
     :open-company-slack-client-secret "FIXME"
-    :aws-access-key "FIXME"
-    :aws-secret-key "FIXME"
```

Comment:
These are the words used by AWS `access-key` & `secret-key`, I don't see the point of changing them here?